### DISCUSSION TITLE
Calendar is showing 12hrs instead of 24

### DISCUSSION THREAD

**Discussion by borisservo** [kimai/kimai#4647] [Questions & Answers] [answered]
Hello,

The calendar is showing in the side where we have the hours for guidance in 12hrs instead of 24hrs.

As per the event itself it is in 24hrs, only calendar is not.
Am I missing some configuration?

![image](https://github.com/kimai/kimai/assets/15837056/767ef821-e7e7-4a5f-aa57-af1df07cc202)

**Comment by kevinpapst** [comment]
This is Kimai 1 and not supported any longer.
Besides: your user profile has a 12/24h switch

**Reply by borisservo** [reply]
Hello @kevinpapst ,

Thank you for that I've update Kimai to the latest version from the docker repo.

Another thing is that I don't see that switch in the user profile, according to the documentation I have to change the language (locale) of the user to have 24h calendar view.
I see no radio button or switch to activate the 24h.
But this is confusing, when create a entry it goes for 24h, but in the calendar is 12h.
Apologies i can't figure this out.

**Reply by kevinpapst** [reply]
If you don't see these settings, you are not on the latest version:
<img width="921" alt="Bildschirmfoto 2024-02-21 um 11 11 36" src="https://github.com/kimai/kimai/assets/533162/f289c997-7e4c-430b-8383-b407bfffc604">

**Reply by borisservo** [reply]
<img width="1502" alt="image" src="https://github.com/kimai/kimai/assets/15837056/d9d894e4-034f-45de-8ebe-254099c3bd55">

These are the options I have, shouldn't it be in the calendar 24h? Portugal uses 24h, that's why I don't get it.

**Reply by kevinpapst** [reply]
You are right, that second configuration is brand new and the calendar seems to still take the 12 hour setting from the first setting.
So this is a bug. Thanks for finding!

**Reply by borisservo** [reply]
Hello @kevinpapst,

No problem, should I create a bug or this thread will be changed to it?

**Reply by kevinpapst** [reply, marked as answer]
https://github.com/kimai/kimai/issues/4653

**Reply by borisservo** [reply]
Thank you @kevinpapst.
I will close the discussion since a bug is already open.
